main.go

```diff
@@ -47,14 +47,24 @@ func runMain() error {
 		return err
 	}
 
-	camera := lepton3.New(conf.SPISpeed)
-	err = camera.Open()
-	if err != nil {
-		return err
+	for {
+		camera := lepton3.New(conf.SPISpeed)
+		err = camera.Open()
+		if err != nil {
+			return err
+		}
+		defer camera.Close()
+		camera.SetLogFunc(func(t string) { log.Printf(t) })
+
+		runRecordings(conf, camera)
+		camera.Close()
+		cameraPowerOffOn(conf.PowerPin)
```

Comment:
The error from `cameraPowerOffOn` shouldn't be ignored either. If it returns an error, bubble it up and let the recorder die.